jacodb-analysis/src/test/kotlin/org/jacodb/analysis/impl/IfdsUnusedTest.kt

```diff
@@ -57,7 +57,7 @@ abstract class IfdsUnusedTest : BaseAnalysisTest() {
         )
     }
 
-    @Disabled
+    @Disabled("#220")
```

Comment:
What is 220?

Reply:
Issue number
https://github.com/UnitTestBot/jacodb/issues/220